### PR TITLE
Updates for handling of product projects

### PR DESCRIPTION
**Pull request by labkey-susanh**
#### Rationale
During testing of the incorporation of products into LKSM, a few bugs were founds and adjustments requested.  Here we implement part of the request to show the Project field by default in grids and detail views when there are product projects available.

#### Related Pull Requests
* https://github.com/LabKey/inventory/pull/619
* https://github.com/LabKey/labkey-ui-components/pull/1033
* https://github.com/LabKey/sampleManagement/pull/1411
* https://github.com/LabKey/biologics/pull/1747

#### Changes
* `getProductProjectsCount `-> `hasProductProjects` and adjust to not count notebook as product projects
* Update product configuration page to use a single radio group for less confusion
* Include the container column in picklist default views

### CODE REVIEW

File: api/src/org/labkey/api/data/Container.java [hunk 1]
```diff
@@ -1678,17 +1678,18 @@ public boolean isProductProjectsEnabled()
...
 
         Container project = getProject();
 
         if (project == null)
-            return 0;
+            return false;
 
-        return project.getChildren().size();
+        // need to exclude the notebook folders in particular here
+        return project.getChildren().stream().anyMatch(c -> c.getContainerType().isInFolderNav());
```
Comment (XingY) [inline api/src/org/labkey/api/data/Container.java:1692]: Good point about notebook folders, should ContainerManager.getProductProjectsMetrics be updated to exclude notebook folders as well?

Comment (XingY) [reply api/src/org/labkey/api/data/Container.java:1692]: Also, should we exclude "Support" containers from here?

**Review by labkey-nicka — APPROVED** [review]

File: api/src/org/labkey/api/data/Container.java (hunk 1 above)
Comment (labkey-susanh) [reply api/src/org/labkey/api/data/Container.java:1692]: Good thought about the metrics.   I'm going to verify whether current production deployments use the home project or not and would have the project-looking "Support" project there or not.

Comment (labkey-susanh) [reply api/src/org/labkey/api/data/Container.java:1692]: Solution has been to delete the /Home/support folder altogether as part of the SampleManagerFolderType configuration.

**Pull request merged**